crates/solver/src/driver/solver_settlements.rs

```diff
@@ -26,6 +26,15 @@ pub struct RatedSettlement {
     pub scaled_unsubsidized_fee: BigRational, // In wei.
     pub gas_estimate: U256,                   // In gas units.
     pub gas_price: BigRational,               // In wei per gas unit.
+    pub solver_balance: U256,                 // In wei
+}
+
+impl RatedSettlement {
+    pub fn is_payable(&self) -> bool {
+        let cost = self.gas_estimate.to_big_rational() * &self.gas_price;
+        let multiplier = BigRational::new(3.into(), 1.into());
```

Comment:
Was the 3x factor chosen because it would have successfully resolved the issues during the FTX crash or is this a rough guess?